resources/puppetlabs/lein-ezbake/template/global/ext/fpm.rb

```diff
@@ -189,24 +189,13 @@
     options.systemd_el = 1
   elsif options.operating_system == :el && options.os_version >= 7 # systemd el
     if ! options.is_pe
-      fpm_opts << "--depends tzdata-java"
       options.java =
-        case options.platform_version
-        when 8
-          # rpm on Redhat 7 may not support OR dependencies
-          if options.os_version == 7
-            'java-11-openjdk-headless'
-          elsif options.os_version == 8
-            '(java-17-openjdk-headless or java-11-openjdk-headless)'
-          elsif options.os_version >= 9
-            'java-17-openjdk-headless'
-          else
-            fail "Unrecognized el os version #{options.os_version}"
-          end
-        when 7
+        if options.os_version == 7
           'java-11-openjdk-headless'
+        elsif options.os_version >= 8
+          'java-17-openjdk-headless'
         else
```

Comment:
Even this else is not needed since line 190 already guarantees `os_version` is at least 7. I'll clean that up in a later patch.